### ISSUE TITLE
Use python_requires in setup.py

### ISSUE THREAD

**Issue by dmwyatt** [cztomczak/cefpython#585]
I haven't yet grokked the seemingly complex build tools for this project, but I was just bitten by the fact that cefpython doesn't support python 3.8.

Is there a reason the python version requirements aren't specified in setup.py with `python_requires`?  Interpreter version constraints should be specified there so the project can't be installed in unsupported python versions unless there's some other reason `python_requires` can't be used.

**Comment by cztomczak** [comment]
Cefpython module will not load on an unsupported python version.

The `python_requires` feature is relatively recent according do documentation, so I don't think it was available at the time when last cefpython was released (2 years ago).

**Comment by dmwyatt** [comment]
> Cefpython module will not load on an unsupported python version.

Right, I know that.  My point is that it should not even be installable on an unsupported python version.

> I don't think it was available at the time when last cefpython was released (2 years ago).

`python_requires` was added summer of 2016 to setuptools and pip.


Anyway, it's OK, no project with a limited number of developers can do everything!  I was just checking if there was a technical reason that `python_requires` was not being used.

Is a PR adding `python_requires` something that you'd be interested in seeing?

**Comment by cztomczak** [comment]
Sure, PR is welcome. Thanks.
